phase1/gce/do

```diff
@@ -46,6 +46,8 @@ fetch_kubeconfig() {
 
   if [[ -n "${SSHUSER}" ]];then
     SSHUSER="${SSHUSER}@"
+  elif [[ -n "${USER-}" ]]; then
```

Comment:
I think a colon is missing, did you mean to use "${USER:-}"

Reply:
Actually, both forms work for defaulting with a very subtle difference (when the variable has been set, but to an empty value:

| Case | `echo ${VAR-default}` | `echo ${VAR:-default}`
| --- | --- | ---
| `VAR='pipejakob'` | `pipejakob` | `pipejakob`
| `VAR=''` |  | `default`
| `unset VAR` | `default` | `default`

Since the default value we're providing is just an empty string, I think both the `${VAR-}` form and `${VAR:-}` is just a matter of preference, and will behave the same. If `${USER:-}` is more familiar or easier to read, I can change it anyway.

Reply:
@pipejakob, Thanks for the explanation. I learnt a new thing in bash :)